src/server/scripts/EasternKingdoms/zone_stormwind_city.cpp

```diff
@@ -494,6 +504,58 @@ class conversation_quest_ancient_curses_accept : public ConversationScript
     EventMap _events;
 };
 
+// 130804 - The King's Command Movie Aura
+class spell_the_kings_command_movie_aura : public SpellScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({
+            StormwindCity::Spells::FadeToBlack
```

Comment:
maybe setting using namespace StormwindCity to reduce verbose?

Reply:
But I would need to set it to every function that uses any of these variables. The other option is to add constexpr variables to the class